src/gentropy/config.py

```diff
@@ -609,6 +609,23 @@ class StudyValidationStepConfig(StepConfig):
     _target_: str = "gentropy.study_validation.StudyValidationStep"
 
 
+@dataclass
+class LocusToGeneEvidenceStepConfig(StepConfig):
+    """Configuration of the locus to gene evidence step."""
+
+    session: Any = field(
```

Comment:
Do you need to specify the session in the config? Hail is set to false by default